client/signup/steps/business-add-ons/index.jsx

```diff
@@ -0,0 +1,197 @@
+	};
+
+	componentDidMount() {
+		this.formStateController = new formState.Controller( {
+			fieldNames: [ 'blogEnabled', 'contactForm', 'leadGeneration', 'simplePayments' ],
+			validatorFunction: noop,
+			onNewState: this.setFormState,
+			hideFieldErrorsOnChange: true,
+			initialState: {
+				blogEnabled: { value: false },
+				contactForm: { value: false },
+				leadGeneration: { value: false },
+				simplePayments: { value: false },
+			},
+		} );
+
+		this.setFormState( this.formStateController.getInitialState() );
+	}
+
+	setFormState = state => {
+		this.setState( { form: state } );
+	};
+
+	handleChangeEvent = event => {
+		this.formStateController.handleFieldChange( {
+			name: event.target.name,
+			value: event.target.value,
+		} );
+	};
+
+	handleSubmit = event => {
+		event.preventDefault();
+		const { goToNextStep, stepName, translate, signupDependencies } = this.props;
+
+		const stepOptionInformation = {
+			blog_enabled: formState.getFieldValue( this.state.form, 'blogEnabled' ),
+		};
+
+		const stepHeadstartInformation = {
+			contactForm: formState.getFieldValue( this.state.form, 'contactForm' ),
+			leadGeneration: formState.getFieldValue( this.state.form, 'leadGeneration' ),
+			simplePayments: formState.getFieldValue( this.state.form, 'simplePayments' ),
+		};
+
+		const stepSiteInformation = Object.assign( {}, signupDependencies.siteInformation, {
+			options: Object.assign(
+				{},
+				signupDependencies.siteInformation.options,
+				stepOptionInformation
+			),
+			headstart: Object.assign(
+				{},
+				signupDependencies.siteInformation.headstart,
+				stepHeadstartInformation
+			),
+		} );
+
+		//Submit Step
+		SignupActions.submitSignupStep(
+			{
+				processingMessage: translate( 'Populating your business contact information.' ),
+				stepName: stepName,
+			},
+			[],
+			{
+				siteInformation: Object.assign(
+					{},
+					signupDependencies.siteInformation,
+					stepSiteInformation
+				),
+			}
+		);
+
+		goToNextStep();
+	};
+
+	skipStep = () => this.props.goToNextStep();
+
+	renderBusinessAddOnsStep() {
+		const { translate } = this.props;
+
+		return (
+			<form onSubmit={ this.handleSubmit }>
+				<Card className="business-add-ons__wrapper">
+					<FormFieldset className="business-add-ons__checkboxes">
+						<FormLabel htmlFor="blogEnabled" className="business-add-ons__checkbox-label">
+							<FormInputCheckbox
+								name="blogEnabled"
+								id="blogEnabled"
+								className="business-add-ons__checkbox"
+								onChange={ this.handleChangeEvent }
+								value={ true }
+							/>
+							{ translate( 'A blog' ) }
+						</FormLabel>
+
+						<FormLabel htmlFor="contactForm" className="business-add-ons__checkbox-label">
+							<FormInputCheckbox
+								name="contactForm"
+								id="contactForm"
+								className="business-add-ons__checkbox"
+								onChange={ this.handleChangeEvent }
+								value={ true }
+							/>
+							{ translate( 'A contact form' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **40** times:
`translate( 'Contact Form' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/signup-business-site-setup)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).